plugin.php

```diff
@@ -45,14 +45,22 @@ public function __invoke()
      */
     protected function register()
     {
-        foreach (['acf/include_field_types', 'acf/register_fields'] as $hook) {
-            add_filter($hook, function () {
-                return new PhoneNumberField(
-                    plugin_dir_url(__FILE__) . $this->assetPath,
-                    plugin_dir_path(__FILE__) . $this->assetPath
-                );
-            });
-        }
+        $field = new PhoneNumberField(
+            plugin_dir_url(__FILE__) . $this->assetPath,
+            plugin_dir_path(__FILE__) . $this->assetPath
+        );
+
+        acf_register_field_type($field);
+
+        // Prevent problems with acf_format=standard
+        add_filter(
+            "acf/rest/format_value_for_rest/type={$field->name}",
+            function ($value_formatted, $post_id, $field, $value) {
+                return $value;
```

Comment:
We should go with `return (new PhoneNumber($value))->toArray();` to provide all the fields. @Log1x What do you think?

Reply:
definitely

Reply:
do we need to do any conditionals to check if $value is empty? we could also do a `->isValid()` check on the PhoneNumber instance before converting to array

Reply:
although I guess field validation would already handle that now that I think about it

Reply:
Yeah, you're right but let's do both to be sure.

Reply:
Okay, I think we can revert the changes because "dry". `PhoneNumber` does everything already: https://github.com/Log1x/acf-phone-number/blob/master/src/PhoneNumber.php#L256-L258

Reply:
oh oops 😊 that's fair. as does the construct

Reply:
can just return it outright with `return (new PhoneNumber($value))->toArray()` I think